VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -30,111 +28,124 @@
     static public String CourseData = TAG + ".course_data";
 
     private EnrolledCoursesResponse courseData;
+    private boolean IS_COURSEWARE_ACCESSIBLE = true;
```

Comment:
oh i forgot to fix that. I had plans to make it `static final` initially, but that can't be done.